web_src/css/base.css

```
@@ -3,7 +3,8 @@
   --fonts-proportional: -apple-system, "Segoe UI", system-ui, Roboto, "Helvetica Neue", Arial;
   --fonts-monospace: ui-monospace, SFMono-Regular, "SF Mono", Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace, var(--fonts-emoji);
   --fonts-emoji: "Apple Color Emoji", "Segoe UI Emoji", "Noto Color Emoji", "Twemoji Mozilla";
-  /* font weights - use between 400 and 600 for general purposes */
+  /* font weights - use between 400 and 600 for general purposes. Default "bold(700)" is excessively bold, so Gitea prefers to use "semibold(600)".
+  If user's font system doesn't have proper weight support (eg: Segoe UI on Linux), weight 600 looks like normal(400), the user should fix their font system. */
```

```suggestion
  /* font weights - use between 400 and 600 for general purposes. 700 decisively not in use */
```

I wouldn't write that second sentence into our config, it just doesn't fit there.

---

Will you answer all related "why my font isn't bolding" questions in the future?

---

They can and should search issues/pr history about it.

---

If you have strong preference for not putting this useful knowledge in comment, I won't insist.

However, I do not like unclear comments. "700 is intentionally not in use": what is the intention? Is it not well-supported? Does it make page messy? Does it conflict with other styles? Is it too bold?

---

I will update. I actually found a use case for bold in https://github.com/go-gitea/gitea/pull/24812.